vllm/model_executor/models/__init__.py

```diff
@@ -71,6 +71,9 @@
     "MedusaModel": ("medusa", "Medusa"),
     "EAGLEModel": ("eagle", "EAGLE"),
     "MLPSpeculatorPreTrainedModel": ("mlp_speculator", "MLPSpeculator"),
+    "JambaForCausalLM": ("jamba", "JambaForCausalLM"),
+    "GraniteForCausalLM": ("granite", "GraniteForCausalLM"),
+    "GraniteMoeForCausalLM": ("granitemoe", "GraniteMoeForCausalLM")
```

Comment:
Please move the entries (except for the existing ones under NOTE) to maintain their alphabetical order. You should also update the Supported Models page to include this model.